Tests/XCUITests/BookmarkingTests.swift

```diff
@@ -119,8 +119,8 @@ class BookmarkingTests: BaseTestCase {
         waitForExistence(app.tables["SiteTable"])
         waitForExistence(app.tables["SiteTable"].cells.staticTexts["www.google"], timeout: 5)
         XCTAssertTrue(app.tables["SiteTable"].cells.staticTexts["www.google"].exists)
-        typeOnSearchBar(text: ".com")
-        typeOnSearchBar(text: "\r")
+        app.textFields["address"].typeText(".com")
```

Comment:
@clarmso I updated the test because it was failing with my latest change for the keyboard.

In theory the case that is failing shouldn't happen in real life what is happening is that `typeOnSearchBar(text: )` is searching for the "url" textfield and tapping on it while the "address" textfield is already shown on top of the "url" one so the user will never be able to tap on it, this is causing the "address" textfield to be created again without the existing text making the Test to fail so the change is to type directly on "address". Let me know if that is ok